ivy_models_tests/clip/test_clip.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import ivy
+import pytest
+import numpy as np
+from PIL import Image
+
+from ivy_models import clip, get_processors
+
+
+@pytest.mark.parametrize(
```

Comment:
just one quick thing on tests, the ci resources are quite limited atm, so could you please ref `test_resnet` and try to refactor them to that convention? Thanks!